frb_codegen/src/utils/command_runner.rs

```diff
@@ -90,16 +89,10 @@ pub(crate) fn execute_command<'a>(
     let args_display = args.iter().map(|path| path.to_string_lossy()).join(" ");
     let mut cmd = Command::new(bin);
     cmd.args(args);
-    if log_enabled!(log::Level::Info) {
-        cmd.stdout(Stdio::inherit()).stderr(Stdio::inherit());
-    }
 
     if let Some(current_dir) = current_dir {
         cmd.current_dir(current_dir);
     }
-    if log_enabled!(log::Level::Info) {
-        cmd.stdout(Stdio::inherit()).stderr(Stdio::inherit());
-    }
```

Comment:
👀

Reply:
These two blocks prevent the output to be captured in the output variable.
This means we can't use the command output if we redirect it to the console directly. It is still shown by the log messages

Reply:
Without these changes the stdout variable was always empty

Reply:
I see. curious why we do this: 

        cmd.stdout(Stdio::inherit()).stderr(Stdio::inherit());

exact same code twice? Is it by mistake or deliberate

Reply:
I think twice is a mistake, but it definitely was meant for passing output and error messages to the user.